library/src/main/java/com/google/android/exoplayer/hls/HlsPlaylistParser.java

```diff
@@ -152,18 +153,20 @@ private static HlsMasterPlaylist parseMasterPlaylist(LineIterator iterator, Stri
         String type = HlsParserUtil.parseStringAttr(line, TYPE_ATTR_REGEX, TYPE_ATTR);
         if (SUBTITLES_TYPE.equals(type)) {
           // We assume all subtitles belong to the same group.
-          String name = HlsParserUtil.parseStringAttr(line, NAME_ATTR_REGEX, NAME_ATTR);
+          name = HlsParserUtil.parseStringAttr(line, NAME_ATTR_REGEX, NAME_ATTR);
```

Comment:
Can we just use a local variable for this (could probably be called subtitleName)? Then get rid of L162 (no longer necessary).